internal/service/service.go

```diff
@@ -195,16 +197,25 @@ func AllInOneServerService(
 
 	evt.ConsumeEvents(handler)
 
+	// Register the reconciler to handle entity events
 	rec, err := reconcilers.NewReconciler(store, evt, cryptoEngine, providerManager, repos)
 	if err != nil {
 		return fmt.Errorf("unable to create reconciler: %w", err)
 	}
-
 	evt.ConsumeEvents(rec)
 
+	// Register the installation manager to handle provider installation events
 	im := installations.NewInstallationManager(ghProviders)
 	evt.ConsumeEvents(im)
 
+	// Register the email manager to handle email invitations
+	mailClient, err := email.NewAWSSES(cfg.Email.AWSSES.Sender, cfg.Email.AWSSES.Region)
```

Comment:
This feels like we should only do this block if `cfg.Email.AWSSES.Sender` is non-empty, but I guess we'll do that in the future to allow local development to not need AWS credentials set up.

Reply:
Addressed in https://github.com/stacklok/minder/pull/3783